Updated mobile styling

YazidZidane opened the pull request:
- Added padding on desktop to prevent the page become too wide
- Updated styling on mobile view, added fixed stepper header and selectedCrops bar.
- Updated theming for texts.
- Refactored Mix Seeding Rate Slider to make it closer to the design file.

mikahpinegar commented:
i'm noticing that everything is off by a factor of 100 in the seed tag info step compared to the deployed version

mikahpinegar commented:
also the `Management Impact on Mix`, `Mix Seeding Rate PLS` and `Bulk Seeding Rate` are different. I think this is due to it originally having the Management impact on mix as 0. is that true?

mikahpinegar commented:
i think the coefficients for management impact on mix is broken. it should be using the coefficient from the `Mix Seeding Rate` step. ex 1 for drilled 0.9 for precision etc

YazidZidane commented:
> i'm noticing that the seeds per pound are off by a factor of 100 in the seed tag info step compared to the deployed version

I just checked the code and this is the correct value. Previously all value were calculated to percent so they are 100 times larger.

mikahpinegar commented:
is the `0.50` value for management impact on mix currently hard coded?

mikahpinegar commented:
if i don't enter in acres it crashes the application. can you not allow progression without entering acres?

mikahpinegar commented:
and maybe add a tooltip that says `Please enter the necessary info below.` to the next button if not everything exists

mikahpinegar commented:
also i noticed only steps.css was magrated to .scss. should all the .css files be updated to .scss?

mikahpinegar commented on src/pages/Calculator/Steps/ReviewMix/StripedLabels.js:
```diff
@@ -17,7 +17,7 @@ const StripedLabels = ({ seed, data, labels }) => {
             xs={12}
           >
-            <Grid item sx={{ textAlign: "justify" }} xs={10}>
+            <Grid item sx={{ textAlign: "justify" }} xs={10} pl={1}>
```
we don't want padding or margin in grids anymore right?

YazidZidane replied:
Only margins in Grid would cause offset problems, padding is fine.

mikahpinegar commented on src/pages/Calculator/Steps/ReviewMix/index.js:
```diff
@@ -418,7 +416,7 @@ const ReviewMix = ({ council }) => {
           </Grid>
 
-          <Grid item xs={12}>
+          <Grid item xs={12} pt={"1rem"}>
```
same thing here, padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/MapComponent.js:
```diff
@@ -54,14 +53,10 @@ const MapComponent = ({
-    <Grid xs={12} container className="map-container">
-      <Grid xs={5} item>
+    <Grid container>
+      <Grid xs={2} item p={"10px"}>
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/RegionSelector.js:
```diff
@@ -91,7 +90,7 @@ const RegionSelector = ({
   return (
     <Grid container>
-      <Grid item xs={7} md={10} className="site-condition-container">
+      <Grid item xs={8} md={10} p={"10px"}>
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/form.js:
```diff
@@ -45,7 +44,7 @@ const SiteConditionForm = ({
   return (
     <>
-      <Grid item xs={12} md={6} className="site-condition-form-container">
+      <Grid item xs={12} md={6} p={"10px"}>
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/form.js:
```diff
@@ -57,7 +56,7 @@ const SiteConditionForm = ({
       </Grid>
 
-      <Grid item xs={12} md={6} className="site-condition-form-container">
+      <Grid item xs={12} md={6} p={"10px"}>
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/form.js:
```diff
@@ -69,7 +68,7 @@ const SiteConditionForm = ({
       </Grid>
 
-      <Grid item xs={12} md={6} className="site-condition-form-container">
+      <Grid item xs={12} md={6} p={"10px"}>
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/form.js:
```diff
@@ -84,18 +83,19 @@ const SiteConditionForm = ({
 
-      <Grid item xs={12} md={6} className="site-condition-form-container">
+      <Grid item xs={12} md={6} p={"10px"}>
         <NumberTextField
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SiteCondition/form.js:
```diff
@@ -84,18 +83,19 @@ const SiteConditionForm = ({
       </Grid>
 
-      <Grid item xs={12} md={12} className="site-condition-form-container">
+      <Grid item xs={12} md={12} p={"10px"}>
```
padding in grid

mikahpinegar commented on src/pages/Calculator/Steps/SpeciesSelection/diversity.js:
```diff
@@ -18,33 +20,37 @@ const Diversity = ({ diversitySelected }) => {
+        sx={{
+          height: "10px",
+          border: "#e5e5e5 1px solid",
+          margin: "5px 0",
```
margin in grid

mikahpinegar commented on src/pages/Calculator/index.js:
```diff
@@ -141,47 +169,96 @@ const Calculator = () => {
+      <Grid
+        item
+        xs={12}
+        paddingTop={"0.625rem"}
```
padding in grid

mikahpinegar commented on src/pages/Calculator/index.js:
```diff
@@ -141,47 +169,96 @@ const Calculator = () => {
+              ? {
+                  position: "fixed",
+                  width: "100%",
+                  paddingTop: "90px",
```
padding in grid

mikahpinegar commented on src/pages/Calculator/index.js:
```diff
@@ -141,47 +169,96 @@ const Calculator = () => {
+        sx={
+          matchesSm && !showHeaderLogo
+            ? activeStep === 0
+              ? { paddingTop: "90px" }
```
padding in grid

YazidZidane commented:
> also i noticed only steps.css was magrated to .scss. should all the .css files be updated to .scss?

I have added tooltip to 1st and 2nd steps if the next button is disabled, and removed unnecessary margins in Grid, deleted App.css. Do we need to change index.css to scss? All css files except for this would be deleted in a further pr.

YazidZidane commented:
> i think the coefficients for management impact on mix is broken. it should be using the coefficient from the `Mix Seeding Rate` step. ex 1 for drilled 0.9 for precision etc

The Seeding Method page seems not updating the managementImpactonMix redux value, only updating a seedingMethod string. We should create a issue for this.
